act/io/armfiles.py

```diff
@@ -254,6 +269,95 @@ def read_netcdf(
     return ds
 
 
+def keep_variables_to_drop_variables(
+        filenames,
+        keep_variables,
+        drop_variables=None):
+    """
+    Returns a list of variable names to use with drop_variables when calling
+    `Xarray.open_dataset` by giving a list of variables to keep. This can
+    greatly help reduce loading time and disk space of the Dataset.
+
+    Will open the netCDF file with netCDF4 library to get list of variable
```

Comment:
Can you clarify this will only work with netcdf files here? I know that you mention that it is using the netCDF4 library, but I think adding that additional clarification would help